Surface_mesh_simplification/include/CGAL/Surface_mesh_simplification/internal/Edge_collapse.h

```diff
@@ -425,7 +425,9 @@ EdgeCollapse(Triangle_mesh& tmesh,
     m_get_placement(get_placement),
     m_should_ignore(should_ignore),
     m_visitor(visitor),
-    m_has_border(!is_closed(tmesh))
+    m_has_border(!is_closed(tmesh)),
+    m_initial_edge_count(0),
+    m_current_edge_count(0)
 {
   m_max_dihedral_angle_squared_cos = CGAL::square(std::cos(1.0 * CGAL_PI / 180.0));
```

Comment:
Shouldn't this also go into the initializer list?

Reply:
`m_max_dihedral_angle_squared_cos`? yes maybe, but at least it is initialized, and the focus of this PR was to initialize the members that aren't

Reply:
Id probably do:
```c++
constexpr cos_1degree_squared = 0.99969541350954786500312172002196498;
//...
m_max_dihedral_angle_squared_cos(cos_1degree_squared)
```